app/views/layouts/comfy/admin/cms.html.haml

```diff
@@ -9,8 +9,9 @@
   -# this is a hack until I figure out how to get chartkick working without importing the whole application js pack
   - if controller_name == 'dashboard'
     = javascript_pack_tag 'application', 'data-turbo-track': 'reload'
-
-
+    %script{type: "text/javascript", src: "//cdn.jsdelivr.net/momentjs/latest/moment.min.js"}
```

Comment:
@Pralish can we install these instead of pulling the latest from a CDN? We are vulnerable to breakages due to changes with the current implementation